src/Core.zig

```diff
@@ -266,6 +265,7 @@ pub fn main(core: *Core, core_mod: mach.Mod(Core)) !void {
     // The user wants mach.Core to take control of the main loop.
     if (supports_non_blocking) {
         while (core.state != .exited) {
+            try Platform.tick(core);
```

Comment:
Nice, this is probably the fix you needed for Linux @joshua-holmes to get the Linux.zig implementation to have a place to poll for events

Reply:
Merging this PR now because of this change.